domains/blockbreaker.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheBlockbreaker",
+        "email": "burstsgodly@gmail.com"
+    },
+    "record": {
+        "TXT": "did=did:plc:agavl34uosnej4h54haas6aj"
```

Comment:
TXT record by itself is useless.

Reply:
I need the TXT record to connect it to bluesky for a custom domain.

Reply:
@TheBlockbreaker We do not support this.